packages/astro/src/client/react/utils.tsx

```diff
@@ -0,0 +1,39 @@
+import type { LoadedClerk } from '@clerk/types';
```

Comment:
❓ also copy pasted?

Reply:
Yes, but `withClerk` is a custom implementation.